sdks/python/apache_beam/ml/inference/base_test.py

```diff
@@ -48,13 +48,40 @@ def run_inference(
       self,
       batch: Sequence[int],
       model: FakeModel,
-      inference_args=None) -> Iterable[int]:
+      inference_args=None,
+      drop_example=False) -> Iterable[int]:
     if self._fake_clock:
       self._fake_clock.current_time_ns += 3_000_000  # 3 milliseconds
     for example in batch:
       yield model.predict(example)
 
 
+class FakeModelHandlerWithPredictionResult(base.ModelHandler[int,
```

Comment:
The name of this class was initially confusing to me. I think I see what you mean since `run_inference` returns `PredictionResult`, but I'm wondering if there's a better way to word this. Maybe `FakeModelHandlerReturnsPredictionResult`?

Reply:
Agreed